### DISCUSSION TITLE
Hashtag '#' appears while hovering on the link.

### DISCUSSION THREAD

**Discussion by Akash3121** [mdn/mdn-community#512] [Content] [answered]
While hovering on the "the logical and operator", there is a hashtag appearing at the end.

Link: https://developer.mozilla.org/en-US/docs/Glossary/Falsy#the_logical_and_operator

<img width="409" alt="hashtag - hover" src="https://user-images.githubusercontent.com/87650180/217566787-873de146-06b0-46d3-930f-71ff67a8052b.png">

https://developer.mozilla.org/en-US/docs/Glossary/Falsy

**Comment by Josh-Cena** [comment, marked as answer]
This is a feature. The hash indicates that it's a hash link.

**Comment by Akash3121** [comment]
Thank you @Josh-Cena. It really helped.
